### DISCUSSION TITLE
How to move avatar?

### DISCUSSION THREAD

**Discussion by ReenigneArcher** [daattali/beautiful-jekyll#904] [Q&A]
How can I move the avatar? My navbar is getting too full and overlapping with the avatar.

![image](https://user-images.githubusercontent.com/42013603/142747680-0becef23-ca51-42b3-83a3-95bcfbef648f.png)

![image](https://user-images.githubusercontent.com/42013603/142747697-fa3ada70-3f3a-4fec-bc27-d91673f77785.png)

**Comment by daattali** [comment]
There is no built-in option to do that. You'll have to either remove the avatar, or (if you're technically inclined) you could add CSS that targets the avatar and moved it to the left. If you don't know anything about CSS then this will be difficult for you, but if you do then I think these two rules should help get you started: `avatar-container { left: 0; } avatar-img-border { margin-left: 0; }`

**Reply by ReenigneArcher** [reply]
Thanks! I actually ended up using the `title-img` attribute in the `_config.yml`. Not quite as pretty but works fine for my purposes.

**Reply by ReenigneArcher** [reply]
I decided to revisit this as there is a slight issue with using the `title-img` alone. Basically their is no icon included with website previews when the site is shared on social media sites or the like. Also it just doesn't look quite as nice.

So changing the css as you suggest does work! However when using the avatar instead of `title-img` the `toggler` is overlapped by the avatar.

Looks great!
![image](https://user-images.githubusercontent.com/42013603/150654878-12be682d-7f26-4ab3-9dd6-a9d3a58bc728.png)

Overlap...
![image](https://user-images.githubusercontent.com/42013603/150654899-72bf77fc-1374-4140-bad3-a9406084d78c.png)

I attempted to move the `navbar-toggler` to the right, but it doesn't seem to be respecting this.
```css
.navbar-toggler {
  font-size: 1rem;
  margin: 0.5rem 0;
  right: 0.9375rem;
}
```

When using the `title-img` the toggler shows on the right side, which is great.
![image](https://user-images.githubusercontent.com/42013603/150655023-893870e2-0870-481c-b83c-d9bacbf5dd3d.png)

Anyway to force the button to be on the right side when not using the `title-img`? Maybe I have the wrong class?
